backend/database_self_healing_system.py

```
@@ -2747,6 +2747,111 @@ async def startup_event():
+            return {"status": "error", "message": "System not initialized"}
+            
+    async def get_system_status(self) -> Dict[str, Any]:
+        """Get current system status"""
+        return {
+            "initialized": self.initialized,
+            "orchestrator_available": self.orchestrator is not None,
+            "status": "healthy" if self.initialized else "not_initialized"
+        }
+
+# Utility function for extracting table names from queries
+def extract_table_from_query(query: str) -> str:
```

Ruff (0.12.2)

Redefinition of unused `extract_table_from_query` from line 144

(F811)